content/pt/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,751 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- thockin
+- msau42
+- xing-yang
+título: Persistentes Volumes
+funcionalidade:
+  title: Orquestração de Storage
+  descrição: >
+    Mountar automaticamente o storage de sua escolha, seja de um storage local, de um provedor de cloud pública, como <a href="https://cloud.google.com/storage/">GCP</a> ou <a href="https://aws.amazon.com/products/storage/">AWS</a>, ou um storage de rede, como NFS, iSCSI, Gluster, Ceph, Cinder ou Flocker.
+
+content_type: conceito
+weight: 20
+---
+
+<!-- visão geral -->
+
+Esse documento descreve o estado atual dos _persistent volumes_ no Kubernetes. Sugerimos que esteja familiarizado com [volumes](/docs/concepts/storage/volumes/).
+
+<!-- body -->
+
+## Introdução
+
+Gerenciamento storage é uma questão bem diferente de gerenciamento de compute instances. O PersistentVolume subsystem provê uma API para usuários e administradores que mostra de forma detalhada de como o storage é provido e como ele é consumido. Para isso, nós introduzidmos duas novas APIs:  PersistentVolume e PersistentVolumeClaim.
```

Comment:
It's better translate "compute instances"?  (maybe to "instâncias de computação")

Reply:
About "compute instances" I'd like to suggest another alternative "instâncias computacionais". What do you think?

Reply:
Cool Emanuel, "instâncias computacionais" is perfect!
Thanks